app/views/trips/dashboard.erb

```diff
@@ -0,0 +1,83 @@
+<h1>trip dash</h1>
+<a href="/trips" style="color:white;font-size:25px;">See All Trips</a>
+<h3><p class="station_title"> average trip duration </p> 1077.21 s</h3>
+<h3><p class="station_title"> longest ride </p> 17270400 s</h3>
+<h3><p class="station_title"> shortest ride </p> 60 s</h3>
+<h3><p class="station_title"> station with most rides as a starting place </p> San Francisco Caltrain (Townsend at 4th)</h3>
+<h3><p class="station_title"> station with most rides as an ending place </p> San Francisco Caltrain (Townsend at 4th)</h3>
+<h3><p class="station_title"> monthly breakdown of ride subtotals by year </p>
+  <% results = Trip.year_iterator %>
+  <table class="table">
+    <tr> 
+      <th> year </th>
+      <th> total </th>
+      <th> jan </th>
+      <th> feb </th>
+      <th> mar </th>
+      <th> apr </th>
+      <th> may </th>
+      <th> jun </th>
+      <th> jul </th>
+      <th> aug </th>
+      <th> sep </th>
+      <th> oct </th>
+      <th> nov </th>
+      <th> dec </th>
+    </tr>
+    <% results.each do |k|%>
+      <tr>
+        <td> <%= k[0] %> </td> 
```

Comment:
this chunk of code is not easy to read. how can we clean this up a bit to make it more readable?